front/app/modules/commercial/smart_groups/components/UserFilterConditions/ValueSelector/EventValuesSelector.tsx

```diff
@@ -13,7 +13,7 @@ export interface Props {
 }
 
 const EventValuesSelector = ({ value, onChange }: Props) => {
-  const { data: events } = useEvents({ sort: '-start_at' });
+  const { data: events } = useEvents({ sort: '-start_at', pageSize: 1000 });
```

Comment:
I increased the page size to 1000 - it was 250 before. This should fix the issue on platforms that have up to 1000 events.